client/reader/search-stream/empty.jsx

```diff
@@ -43,7 +43,7 @@ class SearchEmptyContent extends Component {
 		return (
 			<EmptyContent
 				title={ this.props.translate( 'No results' ) }
-				line={ message }
```

Comment:
We are receiving a `message` from the `translate` function, where we pass components as an argument, which modifies the output of the `translate` function from `string` to `JSX.Fragment`, resulting in different layout.